README.md

```diff
@@ -116,3 +116,6 @@ repository offers more resources to dig into. We are eager to see your first PR!
   * [![Go Reference](https://pkg.go.dev/badge/hey/google)](https://gnolang.github.io/gno/github.com/gnolang/gno.html) \
     (pkg.go.dev will not show our repository as it has a license it doesn't recognise)
 </details>
+
+## Declarations
+* Gno is only available for 64-bit architectures!
```

Comment:
@leohhhn maybe you have a better idea of the right place(s) to add this info?

Reply:
definitely shouldn't go in README; maybe in pkg/gnolang/ readme.

Also we should update goreleaser settings to avoid 32bit targets.